### ISSUE TITLE
Automatic margin reload

### ISSUE THREAD

**Issue by ppumkin** [ppumkin/PrestoCoverage#2]
Figure out how to update the Glyphs without having to open and close the files. 

In debug it enough to start typing but in Release it requires the tab to be closed and opened. 

If any body knows please comment.

**Comment by ppumkin** [comment]
I implemented Filesystem watched to monitor c:\coverlet

I did some more research and switched away from ITaggerProvider to IViewTaggerProvider which gives access to more of the VS IDE stuff.. This stuff will come in handy later.

Whats important is that when you add,remove,rename or update files in the coverlet directory visual studio markers  "Glyphs" update.. and really fast!

Side effects of this
- enables me to manage cache better as I now know which files changed instead of constantly scanning the directory. This should make the caching stuff even more faster
- Kind of helps me to get to the line visits margin aspect as it sets some ground work for new simplified models
